hildon/hildon-touch-selector-entry.c

```diff
@@ -440,13 +448,18 @@ hildon_touch_selector_entry_set_input_mode (HildonTouchSelectorEntry * selector,
 HildonGtkInputMode
 hildon_touch_selector_entry_get_input_mode (HildonTouchSelectorEntry * selector)
 {
+#ifdef MAEMO_GTK
   HildonTouchSelectorEntryPrivate *priv;
 
   g_return_val_if_fail (HILDON_IS_TOUCH_SELECTOR_ENTRY (selector), HILDON_GTK_INPUT_MODE_ALPHA);
 
   priv = HILDON_TOUCH_SELECTOR_ENTRY_GET_PRIVATE (selector);
 
   return hildon_gtk_entry_get_input_mode (GTK_ENTRY (priv->entry));
+#else
+  g_warning("hildon_touch_selector_entry_get_input_mode: no-op, hildon_gtk_entry_get_input_mode unsupported");
```

Comment:
This warning differs from others. Please keep the same style. I mean the comma after "no-op" part.

Reply:
Fixed